docs/GoTopButton.md

```diff
@@ -0,0 +1,39 @@
+📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
+
+# Go Top Button
+
+`GoTopButton` is a component of VTEX that when clicked, goes back to the top of the page. This Component can be imported and used by any VTEX app.
+
```

Comment:
Could you add a media with the `GoTopButton` rendered? It is important for users to see how the app they are installed is actually displayed.